backend/app.js

```diff
@@ -1,21 +1,34 @@
 const express = require("express");
 const path = require("path");
+const mongoose = require("mongoose");
 const app = express();
 
 const indexRouter = require("./routes/indexRouter");
 const chatRouter = require("./routes/chatRouter");
 const authRouter = require("./routes/authRouter");
 
-// Serve static files from the frontend folder globally
 app.use(express.static(path.join(__dirname, "../frontend/public")));
 
+const mongoURI = "mongodb://localhost:27017/myDatabase";
+
+mongoose.connect(mongoURI, { useNewUrlParser: true, useUnifiedTopology: true })
+  .then(() => {
+    console.log("Connected to MongoDB on localhost");
+  })
+  .catch((err) => {
+    console.log("Error connecting to MongoDB:", err);
+  });
```

Comment:
useNewUrlParser and useUnifiedTopology is depreciated. It should look like the following code instead. However, this code should be inside server.js instead.

```
mongoose.connect("mongodb://localhost:27017/db")

Reply:
yes, please move this to server.js. app.js is where we connect to for jest and supertest, server.js (which uses app.js) is where we run the actual server.